downgrade

```diff
@@ -188,6 +213,9 @@ process_term() {
   else
     if present_packages "$term" "${candidates[@]}"; then
       choice=$(read_selection "${candidates[@]}")
+      if [ -z "$choice" ]; then
+        ( gettext "Invalid choice" && echo ) >&2
```

Comment:
I don't think you need these parenthesis. Ditto on L241. (The ones at L295 for example are needed because you're grouping multiple expressions, though `{ ... }` may be better there.)

Reply:
Committed a fix